ui/components/app/assets/hooks/useTokenDisplayInfo.tsx

```diff
@@ -122,7 +122,9 @@ export const useTokenDisplayInfo = ({
     title: token.title,
     tokenImage: token.image,
     primary: formattedPrimary,
-    secondary: token.secondary,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
Do we still need the ts-ignore comment?

Reply:
Actually, i think if i use null instead of undefined, i can then remove this ts ignore 🙏